test/io/test_io.py

```diff
@@ -973,11 +973,10 @@ def test_log_level(level, defaultenv):
                 r'- - postgrest_test_anonymous \[.+\] "GET /unknown HTTP/1.1" 404 - "" "python-requests/.+"',
                 output[2],
             )
+
+            assert len(output) == 5
             assert "Connection" and "is available" in output[3]
-            assert "Connection" and "is available" in output[4]
-            assert "Connection" and "is used" in output[5]
-            assert "Connection" and "is used" in output[6]
-            assert len(output) == 7
+            assert "Connection" and "is used" in output[4]
```

Comment:
Hm, these lines I don't understand. Why does `len(output)` change from 7 to 5?

Reply:
I guess this changed because the output of
```python
response = postgrest.session.get("/unknown")
```
changed in this test. Not sure exactly how it got from 7 to 5.

Reply:
Should be because it's now a PGRST error and it doesn't hit the database anymore (no need to use a connection).